packages/next/pages/_document.tsx

```diff
@@ -389,7 +390,7 @@ export class Head extends Component<
           hasAmphtmlRel = true
         }
       }
-      return child
+      return React.cloneElement(child, { 'data-next-head': true })
```

Comment:
sorta the same concept as `next-head-count` but in-line

Reply:
I don’t think this will work. I think you’ll want to upgrade the `node-html-parser` dep and use that to parse the rendered html, add the data attribute to the top level nodes, then serialize it again. And then your CSR part should work.

Reply:
(To be clear, not 2.0, just the latest minor)

Reply:
the way that person does releases, i'm nervous to even do minor releases! 😂